src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -18,7 +21,7 @@ const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = (props) => {
   const { selectedStudyUIDs } = createAnalysis;
 
   useEffect(() => {
-    const patientInfo = CreateAnalysisService.extractPatientPersonalInfo(dcmImages[0])
+    const patientInfo = CreateAnalysisService.extractPatientPersonalInfo(dcmImages.allDcmImages[0])
```

Comment:
Just as a safety precaution, could we add optional chaining in case there are no DcmImages